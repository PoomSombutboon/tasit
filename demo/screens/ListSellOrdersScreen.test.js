import React from "react";
import NavigationTestUtils from "react-navigation/NavigationTestUtils";
import { shallow } from "enzyme";
import ListSellOrdersScreen from "./ListSellOrdersScreen";

describe("ListSellOrdersScreen", () => {
  jest.useFakeTimers();
  beforeEach(() => {
    NavigationTestUtils.resetInternalState();
  });

  it("renders the component", async () => {
    expect(shallow(<ListSellOrdersScreen />)).toMatchSnapshot();
  });
});
