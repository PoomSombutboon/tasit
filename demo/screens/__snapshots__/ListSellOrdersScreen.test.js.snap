// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListSellOrdersScreen renders the component 1`] = `
<ListLands
  lands={Array []}
  renderRow={[Function]}
/>
`;
